crates/bridge-tx-builder/src/operations.rs

```diff
@@ -281,7 +281,7 @@ mod tests {
             "should work if the number of scripts is not an exact power of 2"
         );
 
-        let secret_key = SecretKey::new(&mut rand::thread_rng());
+        let secret_key = SecretKey::new(&mut rand::rngs::OsRng);
```

Comment:
We should move this to an import
```suggestion
        let secret_key = SecretKey::new(&mut OsRng);
```

Reply:
Agreed. My initial goal was to minimize changes for easier review, but I agree that I prefer imports. Will update.